src/MassTransit/Logging/Monitoring/LogContextInstrumentationExtensions.cs

```diff
@@ -372,21 +372,41 @@ public static class LogContextInstrumentationExtensions
             });
         }
 
+    #if NET8_0_OR_GREATER
+        public static void TryConfigure(IServiceProvider provider)
+        {
+            if (_isConfigured)
+                return;
+
+            var meterFactory = provider.GetService<IMeterFactory>();
```

Comment:
Why? Just default the original behavior if `IMeterFactory` isn't there.

Reply:
why do we need to even create meter? if you haven't subscribed to metrics